docker/ol-infobase-start.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 python --version
-scripts/infobase-server conf/infobase.yml 7000
+scripts/infobase-server "$INFOBASE_OPTS" "$INFOBASE_CONFIG" fastcgi 7000
```

Comment:
- config file needs to go first.
- don't want quotes around `OPTS` since there could be multiple, space-separted options!

```suggestion
scripts/infobase-server "$INFOBASE_CONFIG" $INFOBASE_OPTS 7000
```